packages/bot-web-ui/src/components/dashboard/tutorial-components/tutorial_faq.tsx

```diff
@@ -0,0 +1,375 @@
+import { localize } from '@deriv/translations';
+import { getUrlBase } from '@deriv/shared';
+
+export type TTutorialItem = {
+    title: string;
+    description: { [key: string]: string }[];
+    image?: string;
+};
+
+const getImageLocation = (image_name: string) => getUrlBase(`/public/images/common/${image_name}`);
+
+export const FAQ_QUESTIONS: TTutorialItem[] = [
```

Comment:
aren't we using snake_case format for arrays?

Reply:
Yes. Done